src/web/app/components/loading-spinner/loading-spinner.module.ts

```diff
@@ -1,19 +1,17 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { LoadingSpinnerComponent } from './loading-spinner.component';
+import { LoadingSpinnerDirective } from './loading-spinner.directive';
 
 /**
- * Module for the loading-spinner.
+ * Module for loading spinner
```

Comment:
Just a small nit haha
```suggestion
 * Module for the loading spinner
```